mythril/laser/smt/__init__.py

```diff
@@ -65,24 +75,30 @@ class _SmtSymbolFactory(SymbolFactory):
     """
 
     @staticmethod
-    def Bool(value: bool, annotations=None):
+    def Bool(
+        value: "__builtins__.bool", annotations: Annotations = None
+    ) -> Union[bool.Bool, z3.BoolRef]:
         """
         Creates a Bool with concrete value
         :param value: The boolean value
         :param annotations: The annotations to initialize the bool with
         :return: The freshly created Bool()
         """
-        raw = z3.Bool(value)
+        raw = z3.BoolVal(value)
         return Bool(raw, annotations)
 
     @staticmethod
-    def BitVecVal(value: int, size: int, annotations=None):
+    def BitVecVal(
+        value: int, size: int, annotations: Annotations = None
+    ) -> Union[BitVec, z3.BitVecRef]:
```

Comment:
this too doesn't return z3.BitVecRef right?

Reply:
See above comment for reply.